client/components/theme/more-button.jsx

```diff
@@ -54,7 +54,11 @@ class ThemeMoreButton extends Component {
 
 		return (
 			<span className={ classes }>
-				<button ref={ this.moreButtonRef } onClick={ this.togglePopover }>
+				<button
+					aria-label={ `More options for theme ${ this.props.themeName }` }
```

Comment:
Should this be translated?

Reply:
Yes. This should be translated. I've discovered that most of the alt text on this page, which is also used for screen readers, has not been translated. So, I'm going to make a list and open a translation request for everything I find untranslated on /themes related to a11y outside of this PR.